addons/mail/tests/test_mail_render.py

```diff
@@ -409,6 +409,96 @@ def test_replace_local_links(self):
             rendered = self.env['mail.render.mixin']._replace_local_links(source)
             self.assertEqual(rendered, expected)
 
+    def test_qweb_static_rendering(self):
```

Comment:
Could make a specific test class TestSimplifiedRendering (with a test tag wisely chosen) so that people are able to run tests for this rendering without having to run all other tests maybe ? I quite like having a bit more classes allowing to split a bit big files :)